.travis.yml

```diff
@@ -1,6 +1,14 @@
 language: cpp
 sudo: required
 
+os:
+  - linux
+  - osx
+
+compiler:
+  - gcc
+  - clang
```

Comment:
Yes, that's what I was thinking. But it would need to have an option to configure to clang to use the triple as an argument (and I have very limited knowledge of Waf but I might get there). Anyway, as I said earlier at this point I'm still trying to compile it in native because of a bug in the clang that Travis has (and Travis is really slow to start a build right now...).

Also, this is @8W9aG's work, I might be able to help, but in no way want to "steal" it from him.